spec/factories/taxi_request.rb

```diff
@@ -0,0 +1,14 @@
+FactoryBot.define do
+  factory :taxi_request do
+    driver_id         {  }
+    address           { Faker::Address.full_address }
+    status            { 1 }
+    created_at        { Time.now - 1.months }
+    updated_at        { Time.now - 1.months }
+
+    before(:create) do |taxi_request|
+      passenger = create(:passenger)
+      taxi_request.passenger = passenger
+    end
+  end
```

Comment:
```suggestion
  factory :taxi_request do
    passenger
    driver_id         {  }
    address           { Faker::Address.full_address }
    status            { 1 }
    created_at        { Time.now - 1.months }
    updated_at        { Time.now - 1.months }
  end
```

이렇게 작성하시면 passenger를 create 해서 passenger property에 넣어주어서
before 구문없이 코드를 작성하실 수 있을겁니다!

Reply:
오 유용한 팁이네요 😮 감사합니다!